docs/conf/portland/2020/news/schedule.rst

```diff
@@ -0,0 +1,59 @@
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to have Q&A after most talks
+
+As always, we'll be running the `unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_, and `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch.
+
+You'll be able to sign up for unconference sessions and lightning talks during the conference.
+
+
+Start thinking about Writing Day
+--------------------------------
+
+Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/2019/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!
+
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page, drop us an email at `portland@writethedocs.org <mailto:portland@writethedocs.org>`_.
+Or just show up on Sunday ready to introduce your project!
+
+Shirts on sale
+--------------
+
+The Write the Docs Portland 2019 Pop-Up Shop is now open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want!
+
+The campaign will run in 3-day batches until TODO.
+
+Other events and activities
+----------------------------
+
+Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this years conference is held online, we're holding to that plan.
+
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.
+* **Job Fair**. Following last year's great success, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.
+
```

Comment:
Here we should add something about the social event, badge contest, and the helpdesk maybe?

Reply:
Wanna suggest some copy? <3 @thatdocslady